tests/issues.targets

```diff
@@ -561,8 +561,5 @@
         <ExcludeList Include="$(XunitTestBinBase)\JIT\Regression\CLR-x86-JIT\V2.0-Beta2\b409748\b409748\b409748.cmd">
              <Issue>needs triage</Issue>
         </ExcludeList>
-        <ExcludeList Include="$(XunitTestBinBase)\JIT\opt\Tailcall\TailcallVerifyWithPrefix\TailcallVerifyWithPrefix.cmd">
-             <Issue>needs triage</Issue>
-        </ExcludeList>
     </ItemGroup>
```

Comment:
No, I can't. Several tests in TailcallVerifyWithPrefix have the following sequence:
`tail. call`
`pop`
`ret`
This sequence is illegal according to ECMA-335; however, 64-bit RyuJIT supports it for compatibility with legacy jit. So, the tests pass on x64 and fail with InvalidProgramException on x86.
There is another test that has this sequence: smallFrame so I had to exclude it on x86 as well. The test was passing before my change because the method with this sequence was inlined and tail. prefix was removed.